hooks/useClickOutside.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useRef } from 'react';
+
+type useClickOutsideType = (
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Let's remove this after Hinata's PR has merged

Reply:
Yeah

Reply:
Hmm, I'll cherry-pick the changes to add the rule into another PR tmr so it can be merged first. Because download components migration are blocked by adding the provider first.

Reply:
Done !